src/Uno.UI/OnActivityResultArgs.Android.cs

```diff
@@ -1,26 +1,25 @@
 ﻿using Android.App;
 using Android.Content;
 
-namespace Uno.UI
+namespace Uno.UI;
+
+/// <summary>
+/// OnActivityResultArgs : an object containing the parameters of the Activity.OnActivityResult
+/// </summary>
+/// <param name="ct">CancellationToken</param>
+/// <param name="intent">The Intent you want to send</param>
+/// <param name="requestCode">A specific Response code, this is useful if you send more than one type of request in parallel</param>
+internal class OnActivityResultArgs
```

Comment:
```cs
internal record class OnActivityResultArgs(int requestCode, Result resultCode, Intent intent)
```
?

Reply:
Had to keep the class, as it is currently public in Uno, so it would make it a breaking change...